torch/autograd/functional.py

```diff
@@ -274,7 +275,7 @@ def vjp(func, inputs, v=None, create_graph=False, strict=False):
     return _tuple_postprocess(outputs, is_outputs_tuple), _tuple_postprocess(vjp, is_inputs_tuple)
 
 
-def jvp(func, inputs, v=None, create_graph=False, strict=False):
+def jvp(func, inputs, v=None, create_graph=False, strict=False, fw_mode=False):
```

Comment:
What's the plan for changing the default from False -> True?

Reply:
The plan is to wait until we have more formulas added. This will happen in a follow up of this stack.